inst/app/ui.R

```diff
@@ -123,6 +123,9 @@ ui <- dashboardPage(
               infoBoxOutput(outputId = "infobox_party", width = NULL),
               infoBoxOutput(outputId = "infobox_constituency", width = NULL)
             ),
+            fluidRow(
+              infoBoxOutput(outputId = "individual_page", width = NULL)
```

Comment:
Might be useful to add a [shinyhelper](https://github.com/cwthom/shinyhelper) to describe what they can expect to see from being redirected.